config/passportconfig.js

```diff
@@ -0,0 +1,50 @@
+var passport = require('passport');
+var localStrategy = require('passport-local').Strategy;
+const pool = require('../database/connection');
+
+
+passport.use('local', new localStrategy({ 
+    passReqToCallback : true
+  }, function(req, email, password, done) {
+          console.log('called local strategy');
```

Comment:
Related to #16 
Any ideas @teojunjie?

Reply:
I will remove all console.log after i debug everything